package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
```

Comment:
@kuangp could you bump the package version to `1.0.0` with this PR?
I will merge and release with that change

Reply:
done! thanks!